index.js

```diff
@@ -18,12 +20,21 @@ var splashUrl = 'file://' +
 
 if (parsed.config) {
   process.env.XPY_CONFIG = parsed.config
+} else if (parsed.datadir){
+  process.env.XPY_CONFIG = parsed.datadir+"/paycoin.conf"
 }
 
 var daemon = require('./lib/daemon')()
 
 if (!parsed.daemon && (process.platform === 'darwin' || process.platform === 'linux')) {
-  daemon.start()
+  if (parsed.config) {
+      daemon.start(parsed.config, 0)
+  }
```

Comment:
Remove this extra carriage return please `} else if (parsed.datadir) {`

thanks